src/image_int.hpp

```diff
@@ -5,9 +5,11 @@
 
 // *****************************************************************************
 // included header files
-#include "types.hpp"
+#include "slice.hpp"  // for Slice
 
-// + standard includes
+#include <stddef.h>   // for size_t
```

Comment:
This is something I don't like about blindly using IWYU. ```<cstddef>``` is what should be included. But for some reason IWYU includes the C header. One workaround for this that I've had some success with is replacing size_t with std::size_t for example.

Reply:
Right, I was not careful about this. I also prefer the usage of the <cXXX> headers. I will replace any occurrence of the C-Style headers by the C++ ones.